File: mog-rs/factory/postgres-tables-to-csharp/tests/expected.cs
public class Actor
{
    public int ActorId { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public DateTime LastUpdate { get; set; }
}

public class Staff
{
    public int StaffId { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public int AddressId { get; set; }
    public string? Email { get; set; }
    public int StoreId { get; set; }
    public bool Active { get; set; }
    public string Username { get; set; }
    public string? Password { get; set; }
    public DateTime LastUpdate { get; set; }
    public byte[]? Picture { get; set; }
}

public class Widget
{
    public long WidgetId { get; set; }
    public short SmallCount { get; set; }
    public long? BigCount { get; set; }
    public decimal UnitPrice { get; set; }
    public float? Ratio { get; set; }
    public double? PreciseRatio { get; set; }
    public string Sku { get; set; }
    public string? Grade { get; set; }
    public string? Description { get; set; }
    public bool IsActive { get; set; }
    public DateOnly CreatedOn { get; set; }
    public TimeOnly? OpenTime { get; set; }
    public DateTime MadeAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public Guid ExtId { get; set; }
    public string? Payload { get; set; }
    public string? Meta { get; set; }
    public byte[]? Thumbnail { get; set; }
    public string? Rating { get; set; } // TODO(mog): Postgres enum; consider a C# enum instead of string.
    public string? Fulltext { get; set; } // TODO(mog): Postgres full-search vector; stored as string, not searchable in C#.
    public string[]? SpecialFeatures { get; set; } // TODO(mog): Postgres array; verify element mapping and chosen C# collection.
}

File: mog-rs/factory/postgres-tables-to-python-dataclass/tests/expected.py
from __future__ import annotations
from dataclasses import dataclass
from datetime import date, datetime, time
from decimal import Decimal
from uuid import UUID

@dataclass
class Actor:
    actor_id: int
    first_name: str
    last_name: str
    last_update: datetime

@dataclass
class Staff:
    staff_id: int
    first_name: str
    last_name: str
    address_id: int
    email: str | None
    store_id: int
    active: bool
    username: str
    password: str | None
    last_update: datetime
    picture: bytes | None

@dataclass
class Widget:
    widget_id: int
    small_count: int
    big_count: int | None
    unit_price: Decimal
    ratio: float | None
    precise_ratio: float | None
    sku: str
    grade: str | None
    description: str | None
    is_active: bool
    created_on: date
    open_time: time | None
    made_at: datetime
    updated_at: datetime
    ext_id: UUID
    payload: dict | None
    meta: dict | None
    thumbnail: bytes | None
    rating: str | None # TODO(mog): Postgres enum; consider a Python Enum instead of a plain string.
    fulltext: str | None # TODO(mog): Postgres full-search vector; stored as str, not searchable in Python.
    special_features: list[str] | None # TODO(mog): Postgres array; verify element mapping and chosen Python collection.
